config/project/entryTypes/job--c2bd9b72-0968-4b24-aee2-02048ebf7f8c.yaml

```diff
@@ -0,0 +1,362 @@
+            autocorrect: true
+            class: null
+            dateAdded: '2025-03-13T10:10:32+00:00'
+            disabled: false
+            elementCondition: null
+            id: null
+            includeInCards: false
+            inputType: null
+            instructions: null
+            label: null
+            max: null
+            min: null
+            name: null
+            orientation: null
+            placeholder: null
+            providesThumbs: false
+            readonly: false
+            required: true
+            size: null
+            step: null
+            tip: null
+            title: null
+            type: craft\fieldlayoutelements\entries\EntryTitleField
+            uid: 2f6619fb-d7c5-4567-8252-b6267815e305
+            userCondition: null
+            warning: null
+            width: 100
+          -
+            dateAdded: '2025-04-24T04:24:56+00:00'
+            elementCondition: null
+            fieldUid: 386721be-1ee4-4e3b-b22c-7fb3dfded3ff # Simple Text
+            handle: introduction
+            includeInCards: false
+            instructions: null
+            label: Introduction
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 5649d655-ee29-4868-9d33-6ee23389b42c
+            userCondition: null
+            warning: null
+            width: 100
+          -
+            dateAdded: '2025-03-13T10:20:28+00:00'
+            elementCondition: null
+            fieldUid: a9736c79-b219-4814-ba33-728835b7d230 # Body
+            handle: null
+            includeInCards: false
+            instructions: null
+            label: Description
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 5decdee0-6442-4b95-9fd7-63c85b8648fc
+            userCondition: null
+            warning: null
+            width: 100
+        name: Page
+        uid: 2591bb92-334d-429f-af49-cc1781cb0df4
+        userCondition: null
+      -
+        elementCondition: null
+        elements:
+          -
+            dateAdded: '2025-04-24T10:47:36+00:00'
+            elementCondition: null
+            fieldUid: a6b97e16-5236-44b1-ad57-3899644efd40 # Link (Assets, Entries, URL)
+            handle: cta
+            includeInCards: false
+            instructions: null
+            label: 'Application / Call To Action'
+            providesThumbs: false
+            required: false
+            tip: null
+            type: craft\fieldlayoutelements\CustomField
+            uid: 2e31b2a2-6b3e-4618-8257-603ce8ce25fa
+            userCondition: null
+            warning: null
+            width: 100
+          -
+            dateAdded: '2025-03-13T10:20:28+00:00'
+            elementCondition: null
+            fieldUid: 32ed2f1a-9024-4ee9-9028-3a5041c0e5e6 # Date
+            handle: null
```

Comment:
Should use `deadline` as the field handle to be more semantic about the data being captured.